src/Components/Layout/Navbar.tsx

```diff
@@ -1,44 +1,75 @@
 import React from "react"
-import { 
+import {
   HStack,
+  IconButton,
   Image,
   Flex,
   Link,
   LinkBox,
   LinkOverlay,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
 } from '@chakra-ui/react'
 import { Link as GatsbyLink } from 'gatsby'
+import { HamburgerIcon } from '@chakra-ui/icons'
 
 export const Navbar = (): JSX.Element => {
   return (
-    <Flex alignItems='center' bgColor='purple' px={16} py={12} justifyContent='space-between'>
+    <Flex alignItems='center' bgColor='purple' px={['24px', '64px']} py={['24px', '48px']} justifyContent='space-between'>
       <LinkBox as={HStack}>
         <Image src='../../../Images/Logos/ColorLogo.jpg' boxSize='45px'/>
         <LinkOverlay 
           as={GatsbyLink}
           to='/'
           color='white'
-          fontSize={32}
+          fontFamily='Helvetica'
+          fontSize={[24, 32]}
           fontWeight='bold'
           _hover={{
             textDecoration: 'underline',
             textDecorationColor: 'orange',
           }}
         >
-          Internet for All NYC
+          Internet for All
         </LinkOverlay>
       </LinkBox>
-      <HStack spacing={6}>
-        <Link as={GatsbyLink} to='/principles' variant='nav'>
+      <HStack spacing='24px' display={['none', 'block']}>
```

Comment:
This is [Chakra's array syntax for responsive styles](https://chakra-ui.com/docs/features/responsive-styles#the-array-syntax) that cause the links not to show in a horizontal row on the smallest breakpoint (corresponding to our mobile design)